rotkehlchen/chain/ethereum/interfaces/ammswap/ammswap.py

```diff
@@ -0,0 +1,991 @@
+            for _, trades in trade_mapping.items():
+                all_trades.extend(trades)
+
+            return all_trades
+
+    def _fetch_trades_from_db(
+            self,
+            addresses: List[ChecksumEthAddress],
+            from_timestamp: Timestamp,
+            to_timestamp: Timestamp,
+    ) -> AddressTrades:
+        """Fetch all DB AMM trades within the time range"""
+        db_address_trades: AddressTrades = {}
+        for address in addresses:
+            db_swaps = self.database.get_amm_swaps(
+                from_ts=from_timestamp,
+                to_ts=to_timestamp,
+                location=self.location,
+                address=address,
+            )
+            db_trades = self.swaps_to_trades(db_swaps)
+            if db_trades:
+                db_address_trades[address] = db_trades
+
+        return db_address_trades
+
+    def _get_balances_graph(
+            self,
+            addresses: List[ChecksumEthAddress],
+    ) -> ProtocolBalance:
+        """Get the addresses' pools data querying the Sushiswap subgraph
+
+        Each liquidity position is converted into a <LiquidityPool>.
+        """
+        address_balances: DDAddressToLPBalances = defaultdict(list)
+        known_assets: Set[EthereumToken] = set()
+        unknown_assets: Set[EthereumToken] = set()
+
+        addresses_lower = [address.lower() for address in addresses]
+        querystr = format_query_indentation(LIQUIDITY_POSITIONS_QUERY.format())
+        param_types = {
+            '$limit': 'Int!',
+            '$offset': 'Int!',
+            '$addresses': '[String!]',
+            '$balance': 'BigDecimal!',
+        }
+        param_values = {
+            'limit': GRAPH_QUERY_LIMIT,
+            'offset': 0,
+            'addresses': addresses_lower,
+            'balance': '0',
+        }
+        while True:
+            try:
+                result = self.graph.query(
+                    querystr=querystr,
+                    param_types=param_types,
+                    param_values=param_values,
+                )
+            except RemoteError as e:
+                self.msg_aggregator.add_error(
+                    SUBGRAPH_REMOTE_ERROR_MSG.format(
+                        error_msg=str(e),
+                        location=self.location,
+                    ),
+                )
+                raise
+
+            result_data = result['liquidityPositions']
+            for lp in result_data:
+                lp_pair = lp['pair']
+                lp_total_supply = FVal(lp_pair['totalSupply'])
+                user_lp_balance = FVal(lp['liquidityTokenBalance'])
+                try:
+                    user_address = deserialize_ethereum_address(lp['user']['id'])
+                    lp_address = deserialize_ethereum_address(lp_pair['id'])
+                except DeserializationError as e:
+                    msg = (
+                        f'Failed to Deserialize address. Skipping pool {lp_pair} '
```

Comment:
need to add location here
```suggestion
                        f'Failed to Deserialize address. Skipping {self.location} pool {lp_pair} '
```